app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -109,6 +119,17 @@ def parse_build_pod(build_pod)
                                                            build_pod.status.config.try(:name))
         new_result
       end
+
+      def parse_openshift_image(openshift_image)
+        docker_json = JSON.parse(openshift_image[:dockerImageManifest])
+        name = docker_json["name"]
+        tag = docker_json["tag"]
+        image_name = "#{name}:#{tag}"
+        image_id = "docker://%s" % openshift_image[:metadata][:name]
```

Comment:
Yes, The imageID looks like that in the container definition that we used so far too[1]
Upon more examination I understand now that they have many different "ID"s and names there and I want to look at it again to find the right one...

@miq-bot add_label wip

https://gist.github.com/enoodle/acbdc49b73804dec3004f9c34bf619fa#file-gistfile1-txt-L135